src/test/java/org/fundacionjala/trello/hooks/TeamHooks.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.trello.hooks;
+
+import io.cucumber.java.After;
+import org.fundacionjala.trello.client.RequestManager;
+import org.fundacionjala.trello.context.Context;
+
+import static org.fundacionjala.trello.context.EndPointsEnum.TEAM;
+
+public class TeamHooks {
+
+    private static final int CLEAN_CONTEXT_ORDER_VALUE_TEAM = 10;
```

Comment:
this value may overlap with hooks defined on common hooks.
- maybe getting this one to 20(and subsequently the board hook to 30)
- or assigning this one to 15 
- or getting the common hooks on lower numbers

Reply:
You are right.

Reply:
But I would like to reduce the @after of CommonHooks to 5.

Reply:
That's better, also the INIT and CLEAN values could be the same(as these are related to the open and close of the driver)

Reply:
It's done